Our gtksourceview already have a vbnet.lang and nemerle.lang
Alejandro

--- ./Makefile.am.orig	2005-02-15 10:11:54.538525128 -0300
+++ ./Makefile.am	2005-02-15 10:32:22.066912448 -0300
@@ -1,9 +1,9 @@
 
 SUBDIRS=		gtksourceview sample doc 
-EXTRA_DIST=		nemerle.lang vbnet.lang gtksourceview-sharp.pc.in gtk-sharp.snk
+EXTRA_DIST=		gtksourceview-sharp.pc.in gtk-sharp.snk
 
 extra_langdir= $(gtksourceview_prefix)/share/gtksourceview-1.0/language-specs
-extra_lang_DATA=	nemerle.lang vbnet.lang
+extra_lang_DATA=	
 
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = gtksourceview-sharp.pc
